assets/dev/js/admin/admin.js

```diff
@@ -242,6 +242,32 @@
 				} ).show();
 			} );
 
+			$( '.elementor-re-migrate-globals-button' ).on( 'click', function( event ) {
```

Comment:
1. At least for newly added code, use caching
2. Use arrow function

Reply:
1. Done.
2. The `this` should be referred to the event target.

Reply:
You can use `event.currentTarget` to achieve this

Reply:
Done.